rtc/introduction.md

```diff
@@ -70,6 +70,26 @@ Here is a list of plugins currently supported:
 * [keep_styles]({{site.baseurl}}/configure/content-filtering/#keep_styles)
 * [resize_img_proportional]({{site.baseurl}}/configure/advanced-editing-behavior/#resize_img_proportional)
 * [valid_children]({{site.baseurl}}/configure/content-filtering/#valid_children)
+* [allow_unsafe_link_target]({{site.baseurl}}/configure/content-filtering/#allow_unsafe_link_target)
```

Comment:
It might be worthwhile putting this list in some sort of meaningful order (alphabetically?) as it's getting quite long now :shrug: